GVRf/Framework/framework/src/main/java/org/gearvrf/x3d/X3Dobject.java

```diff
@@ -1036,6 +1036,7 @@ else if (qName.equalsIgnoreCase("ImageTexture")) {
                     gvrTextureParameters = new GVRTextureParameters(gvrContext);
                     gvrTextureParameters.setWrapSType(TextureWrapType.GL_REPEAT);
                     gvrTextureParameters.setWrapTType(TextureWrapType.GL_REPEAT);
+                    gvrTextureParameters.setMinFilterType(GVRTextureParameters.TextureFilterType.GL_LINEAR_MIPMAP_NEAREST);
```

Comment:
did we test this with compressed textures? filtering says we are using mip-mapping.